pkgs/applications/audio/gradio/default.nix

```diff
@@ -51,15 +48,20 @@ in stdenv.mkDerivation rec {
 
     wrapGAppsHook
     desktop-file-utils
-    gsettings-desktop-schemas
-  ] ++ gst_plugins;
+  ];
+
+  patchPhase = ''
+    # Disable the custom build script to run it in post-install phase
+    sed -i "/meson.add_install_script('meson_post_install.sh')/d" ./meson.build
+  '';
 
-  enableParallelBuilding = true;
   postInstall = ''
+    # Running commands from disabled in patchPhase meson_post_install.sh
+    gtk-update-icon-cache -qtf "$out"/share/icons/hicolor
```

Comment:
We do not need these two on Nix. Actually, we are trying to avoid them.

Reply:
Done